lib/nostrex/events/event.ex

```diff
@@ -98,18 +99,39 @@ defmodule Nostrex.Events.Event do
   """
   def calculate_id(event) do
     {:ok, serialized_event} = serialize(event)
-
-    :crypto.hash(:sha256, serialized_event)
+    serialized_event
+    |> Bitcoinex.Utils.sha256()
```

Comment:
why use Bitcoinex here?

Reply:
I thought it would be simpler to have Bitcoinex handle all the crypto-related functions. Should we switch it back?